runtime/cmake/platform.cmake

```diff
@@ -20,57 +20,18 @@
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 ################################################################################
 
+# Get OMR's platform config
 include(OmrPlatform)
-# Note: we need to inject WIN32 et al, as OMR no longer uses them
-if(OMR_OS_WINDOWS)
-    list(APPEND OMR_PLATFORM_DEFINITIONS
-        -DWIN32
-        -D_WIN32
-    )
-    if(OMR_ENV_DATA64)
-        list(APPEND OMR_PLATFORM_DEFINITIONS
-            -DWIN64
-            -D_WIN64
-        )
-    endif()
-endif()
-omr_platform_global_setup()
 
-if(OMR_TOOLCONFIG STREQUAL "gnu")
-    set(CMAKE_CXX_FLAGS " -g -fno-rtti -fno-exceptions ${CMAKE_CXX_FLAGS}")
-    set(CMAKE_C_FLAGS "-g ${CMAKE_C_FLAGS}")
-
-    # Raise an error if a shared library has any unresolved symbols.
-    # This flag isn't supported on OSX, but it has this behaviour by default
-    if(NOT OMR_OS_OSX)
-        set(CMAKE_SHARED_LINKER_FLAGS "-Wl,-z,defs ${CMAKE_SHARED_LINKER_FLAGS}")
-    endif()
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -pthread -O3 -fno-strict-aliasing")
```

Comment:
Is -pthread also inherited from omr?

Reply:
yes
https://github.com/eclipse/omr/blob/2053a62aa58722f07365f7783fe7982728bd839f/cmake/modules/platform/toolcfg/gnu.cmake#L86